config/config_test.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import (
+	"github.com/go-yaml/yaml"
+	"testing"
+)
+
+type TestYaml struct {
+	Field YAMLURL
+}
+
+func TestYAMLURLParsing(t *testing.T) {
```

Comment:
Split to two functions with more meaningful test names. Something like `TestIfParsesCorrectYAML`, `TestIfParsesIncorrectYAML`.